src/runtime/go.mod

```diff
@@ -117,9 +118,9 @@ require (
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.45.0 // indirect
 	go.opentelemetry.io/otel/metric v1.19.0 // indirect
 	golang.org/x/mod v0.13.0 // indirect
-	golang.org/x/net v0.33.0 // indirect
+	golang.org/x/net v0.36.0 // indirect
 	golang.org/x/sync v0.6.0 // indirect
-	golang.org/x/text v0.21.0 // indirect
+	golang.org/x/text v0.22.0 // indirect
```

Comment:
x/text itself is not upgraded because of the `replace` directive. Similar to this PR reasoning: https://github.com/kata-containers/kata-containers/pull/11003 ... a better way would be to remove the replace directives for dependencies that are supposed to be updated, then run `make vendor` to get this right.

So, this is problematic because we assum we update x/test but in reality we don't. See, no changes under vendor/golang.org/x/text ....